### PR TITLE
[Logs] PCX-1695: Add dataset name to Log fields generated files

### PR DESCRIPTION
**Pull request by soheiokamoto**
This adds dataset name to Log fields generated files.

Code to generate is being updated.

### CODE REVIEW

**Review by tlozoot — APPROVED** [review]
Looks good but would be good to have @crwaters16 approve

**Review by crwaters16 — APPROVED** [review]

File: products/logs/src/content/reference/log-fields/firewall_events.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ title: Firewall events
 
 # Firewall events
 
+dataset: firewall_events
```
Comment (crwaters16) [inline products/logs/src/content/reference/log-fields/firewall_events.md]: Let's update this to: The descriptions below detail the fields available for `firewall_events`.

That way we have a little more context leading into each page's table.

**Review by crwaters16 — CHANGES_REQUESTED** [review]

File: products/logs/src/content/reference/log-fields/firewall_events.md (hunk 1 above)
Comment (soheiokamoto) [reply products/logs/src/content/reference/log-fields/firewall_events.md]: Updated.

**Review by crwaters16 — APPROVED** [review]
I'm pretty sure this has already been merged in, just cleaning up my github

**Comment by soheiokamoto** [comment]
> I'm pretty sure this has already been merged in, just cleaning up my github

Yeah, I forgot to merge this one, and updated along with this PR: https://github.com/cloudflare/cloudflare-docs/pull/1861

**Pull request closed without merging**